### ISSUE TITLE
Prozesse ohne Bild führen auf INT zu 500er

### ISSUE THREAD

**Issue by larsUE** [decidim-swiss/decidim.swiss#45]
Wenn ich einen Prozess erstelle und kein Bild hinzufüge, crashen alle Seiten, die das Bannerbild darstellen sollten (z.B. /processes). Vermutung: Es wird versucht, das Standardbild der Stadt Luzern zu laden, kann dies aber nicht finden. Wenn ich ein Bild hinzufüge, klappt es ohne Probleme. Nachgestellt auf gemeinsamer.space INT und Co-Kreation INT.

**Comment by Kagemaru** [comment]
Es probiert tatsächlich ein Standardbild zu laden, nämlich das von der Organisation. Da dies nicht hinterlegt ist, gibt es einen 500er.

Soll ich ein Backup einbauen, falls keines vorhanden ist, oder wollt ihr ein Bild bei `Highlighted Inhaltsbanner` hochladen?

**Comment by larsUE** [comment]
Nein, ist gut. Wir laden eins hoch beim Highlighted Inhaltsbanner!
